### PR TITLE
Updated build tool version for NET 5 RC2

### PR DESCRIPTION
**Pull request by nohwnd**

### CODE REVIEW

**Comment by filipw** [comment]
please update the tests too - see https://github.com/OmniSharp/omnisharp-roslyn/pull/1953 for reference

**Comment by JoeRobich** [comment]
I updated the MSBuild pacakging so that all platforms now use the "net472" resolvers.

**Comment by filipw** [comment]
@JoeRobich I am not sure I get the change. so it first copies the Mono ref assemblies from global Mono but then overwrites whatever it can from Nuget, correct?

**Comment by JoeRobich** [comment]
> so it first copies the Mono ref assemblies from global Mono but then overwrites whatever it can from Nuget, correct?

@filipw Yes, it is doing the same thing that master is presently doing. I just refactored so that the common steps are more apparent. Since some targets and ref assemblies are needed from Mono's MSBuild, we copy those over first, since they are not in the MSBuild NuGets. We then overwrite with the version of MSBuild we want to ship with from the NuGet. I can update the list of MSBuild runtime targets and props so that we are only copying over what isn't present in the NuGets.

File: tools/packages.config [hunk 1]
```diff
@@ -3,31 +3,31 @@
     <package id="Cake" version="0.37.0" />
     <package id="Microsoft.Bcl.AsyncInterfaces" version="1.1.1" />
     <package id="Microsoft.Build" version="16.8.0-preview-20429-01" />
```
[the remark this answers is not in the document]
Comment (nohwnd) [reply tools/packages.config]: You are right. I was doing it via a script and this entry was missing. Fixed that script and pushed the change.

**Review by JoeRobich — APPROVED** [review]

**Review by filipw — APPROVED** [review]
thanks!

**Comment by filipw** [comment]
shall we upgrade Roslyn too?

**Comment by JoeRobich** [comment]
> shall we upgrade Roslyn too?

Sure. Let me see what we are likely shipping in 16.8 Preview 4.

**Comment by nohwnd** [comment]
@JoeRobich did I miss more entries in that xml?

**Comment by JoeRobich** [comment]
@nohwnd You didn't miss anything in this PR. Prior to an OmniSharp release it is nice to bump the Roslyn used by the language services as well. see https://github.com/OmniSharp/omnisharp-roslyn/pull/1974

**Pull request merged**